src/main/java/seedu/address/logic/commands/DeleteScoreCommand.java

```diff
@@ -23,13 +23,13 @@ public class DeleteScoreCommand extends Command {
             + "Parameters: INDEX_OF_STUDENT INDEX_OF_SCORE (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1 2";
 
-    public static final String MESSAGE_DELETE_SCORE_SUCCESS = "Deleted Score for %1$s: %2$s";
+    public static final String MESSAGE_DELETE_SCORE_SUCCESS = "Deleted score for %1$s: %2$s";
 
     private final Index studentIndex;
     private final Index scoreIndex;
 
     /**
-     * Creates an deleteScoreCommand to add the specified {@code Person}
+     * Creates a DeleteScoreCommand to delete the specified task from a specified person
```

Comment:
Do we need uppercase letter for `DeleteScoreCommand`?
Maybe instead of task, we may change to score, since this class related to score.
`Creates a deleteScoreCommand to delete the specified score from a specified student`

Reply:
Okay! And yes, we need uppercase letter because we create a DeleteScoreCommand object